cg/store/api/add.py

```diff
@@ -137,15 +136,50 @@ def add_delivery(self, destination: str, sample: models.Sample=None, pool: model
         new_record.pool = pool
         return new_record
 
-    def add_invoice(self, customer: models.Customer, samples: List[models.Sample]=None,
-                    pools: List[models.Pool]=None, comment: str=None, discount: int=0, 
-                    record_type: str=None):
+    def add_invoice(self, customer: models.Customer, samples: List[models.Sample] = None,
+                    pools: List[models.Pool] = None, comment: str = None, discount: int = 0,
+                    record_type: str = None):
         """Build a new Invoice record."""
         new_id = self.new_invoice_id()
-        new_invoice = self.Invoice(comment=comment, discount=discount, id=new_id, record_type=record_type)
+        new_invoice = self.Invoice(comment=comment, discount=discount, id=new_id,
+                                   record_type=record_type)
         new_invoice.customer = customer
         for sample in samples or []:
             new_invoice.samples.append(sample)
         for pool in pools or []:
             new_invoice.pools.append(pool)
         return new_invoice
+
+    def add_microbial_order(self, customer: models.Customer, name: str, ordered: dt.datetime,
+                            internal_id: str = None, ticket_number: int = None,
+                            comment: str = None) -> models.MicrobialOrder:
+        """Build a new Order record."""
+        new_order = self.MicrobialOrder(name=name, ordered_at=ordered, internal_id=internal_id,
+                                        ticket_number=ticket_number, comment=comment)
+        new_order.customer = customer
+        return new_order
+
+    def add_microbial_sample(self, name: str, strain: str, strain_other: str, internal_id: str,
+                             reference_genome: str,
+                             application_version: models.ApplicationVersion,
+                             priority: str = None,
+                             comment: str = None, **kwargs) -> models.MicrobialSample:
+        """Build a new MicrobialSample record.
+        
+        To commit you also need to assign the sample to an Order.
+        """
+        internal_id = internal_id or utils.get_unique_id(self.sample)
+        priority_human = priority or 'standard'
+        db_priority = PRIORITY_MAP[priority_human]
```

Comment:
db_priority = PRIORITY_MAP[priority_human] 
make the priorities consistent regarding order